CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ Releases 4.5.x are based on mainline support submitted in
 
 ## [Unreleased]
 
+## [v4.5.4] - 2017-01-24
+### Changed
+- optimizations for SeaBIOS, faster boot
+- fix `WARNING - Timeout at i8042_flush:71!` message during boot
+- fix `ASSERTION ERROR: file 'src/northbridge/amd/pi/agesawrapper.c', line 305`
+  messager during boot
+- add coreboot build string message during boot
+- fix disabling/enabling iPXE rom by `sortbootorder`
+- fix custom menu/config options for iPXE rom
```

Comment:
@mek-x BTW we need APU3 in legacy also, but this should be needed probably with next release.